TestTransfers/file_format_extensions_only/README.md

```diff
@@ -0,0 +1,24 @@
+## Variously configured files to test handling of file format extensions
+
+The files in these folders are designed to test various different format
+identification tools' (and the format policy register's) ability to look at
+files with only a file-format-extension as its primary signature. Files
+should have an extension known to PRONOM, e.g. `.jpg`, `.bin`, `.class` but
+should not contain characteristics that would identify the file via any
+signature-based method available to the tool. Two folders are set-up with a
+limited number of samples.
+
+### empty_files_with_extensions
+
+To test extension-only identification and not risk identification via a format
+identification tool's binary matching techniques, these files are devoid of
+any content. Their file-sizes should be zero. They will also be free from
+various byte-order-markings used in Unicode-like formats such as UTF-8.
```

Comment:
It seems inaccurate to me to call UTF-8 a "Unicode-like format" since UTF-8 is really one method of encoding Unicode strings. I may not be saying that exactly right or I may be being pedantic, but something seems odd about this so I'm just putting it out there.